### ISSUE TITLE
Add new semantic  sort options to all scope tabs (from 2023)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#80]
**Problem Description:**
New Sorting options are desired for each Object, Works, People/Group, Places, Concepts and Events tab. The new sorting options are  semantic or non semantic (NS)  and can be found [here](https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit?gid=0#gid=0). 

This ticket will be used for all the semantic sort options. 

**Expected Behavior/Solution:**
Object, Works, People/Groups, Concepts and Event tab will contain new semantic  sort options. 
All new options will have an ascending/descending option. 
All options will heavily rely on SortName with the expectation there isn't 100% consistency in the record. 

**Requirements:**
Those which are **highlighted pink** identify the semantic sort options to be added & the order which they should go in: 
https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit?gid=0#gid=0

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.
- [x] Wireframe/Mockup - Heather
- [x] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [x] Frontend requirements- TBD
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
- TST: [Sun](https://lux.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22sun%22%2C%22_lang%22%3A%22en%22%7D&sq=sun)

**Dependencies/Blocks:**
- Blocked by: https://github.com/project-lux/lux-marklogic/issues/343 which is 
- - Blocked by: https://github.com/project-lux/lux-marklogic/issues/390
- - Blocked by: https://github.com/project-lux/lux-marklogic/issues/10 | What happens from this issue could define the sort options in this issue. 

**Related Github Issues:**
- https://github.com/project-lux/lux-frontend/issues/32
- non-semantic: https://github.com/project-lux/lux-frontend/issues/396

**Related links:**
- [Sorting google spreadsheet](https://docs.google.com/spreadsheets/d/1c9kqnSqTVSGSlV4FkX0wXRphwTYlbHuApP500QtQzGU/edit#gid=0)
- [Notes | 2024-09-12: Semantic Search
](https://docs.google.com/document/d/1Y0pxrib4xa9U7U7lIY2CagPTgjSWe0hvAfJ81HC1SbI/edit)

**Wireframe/Mockup:**

**Objects**
![Image](https://github.com/user-attachments/assets/8176c3b7-5fd1-45b1-90b9-d479a0ee5c34)
![Image](https://github.com/user-attachments/assets/3e3d1270-b5fb-4168-8433-5ade66a6d5d1)
**Works**
![Image](https://github.com/user-attachments/assets/572698b5-4bf8-4725-abf3-6a7449808da7)
![Image](https://github.com/user-attachments/assets/817a3363-397a-4c01-be41-801e2de3d367)
**People/Org**
![Image](https://github.com/user-attachments/assets/bf7abdc2-b8d2-4f1e-8c0b-79fd343b0ae8)
![Image](https://github.com/user-attachments/assets/1d5fa429-4829-4e15-9042-34d0ab00fbb9)\
**Places**
![Image](https://github.com/user-attachments/assets/c21f3bbe-6b6b-4f98-8b2f-4287b7e68935)
**Concepts**
![Image](https://github.com/user-attachments/assets/a195e0b5-3e32-480c-8895-aae4939de321)
**Events**
![Image](https://github.com/user-attachments/assets/dbe16172-cc1a-4b86-b485-8fedd1776863)
![Image](https://github.com/user-attachments/assets/8527fff2-2a23-41ff-854c-210624151426)

**Comment by roamye** [comment]
Awaiting response from https://github.com/project-lux/lux-marklogic/issues/10#issuecomment-2248887831 to determine when this ticket can move forward.

**Comment by prowns** [comment]
Per 8/20/24 UX/BA mtg, @roamye will update spreadsheet with current sort options and review with @dcheather and team to see if there are other semantic sort options to add across the system.

related tix: FE 32 and ML 10.

**Comment by roamye** [comment]
This has been updated in the spreadsheet above. The updated version is at the top. I kept the old at the bottom in the case it brings inspiration to new semantic sorts we would like to add.

**Comment by roamye** [comment]
Proposal to make this issue the meta ticket. I have listed above the new additions for each tab which can also be found in the spreadsheet. 

Q: Should we make a new ticket for each addition? Or are there additions that can be grouped together in one ticket? 

Ex: Categorized and Class' additions become one ticket that will be applied to all scopes. Rather than 5 tickets for the addition of Categorized within Work, Place, etc...

Thoughts? 
@kamerynB @prowns

**Comment by kamerynB** [comment]
@roamye The bulk of this work will take place in ML with the frontend only requiring the addition of the sorting options. Ultimately, the frontend should be able to add all sorting options in one issue, or 6 issues, one issue per scope (Objects, works, people, etc). I would ask @clarkepeterf  about the ML work. I would also be available for helping Peter with the ML work if needed.

**Comment by roamye** [comment]
From UX/BA meeting 9/25: 
- Heather has the mockups ready but an order needs to be defined first. 
- The sorting order has been updated to include Relevance as the first sort, Name/Title as the second sort option and shuffle as the last sort option. All other sort options will be ordered by alphabet.
- This will be brought to UAT to confirm the order.

--- 

@clarkepeterf - from Kam's comment above, would the ML work fall into one ticket? Also what kind of ML work is needed for this?

**Comment by roamye** [comment]
UAT 9/27: 
- Next steps; 
- Order has been approved. 
- Offer all the sort options and this can be further reviewed.

**Comment by miketullo95** [comment]
wireframes updated @roamye

**Comment by roamye** [comment]
unblocking this issue as https://github.com/project-lux/lux-marklogic/issues/10 is closed.

**Comment by roamye** [comment]
@clarkepeterf - 
Objects: 
- Collection and Responsible Unit are not available within the sort | [example](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22cat%22%2C%22_lang%22%3A%22en%22%7D&sq=cat)
- Order of options is not correct. 

Works + People/Orgs + Concepts + Events: 
- order of options is not correct

Places: 
- Order of options is not correct. 
- Cannot find `by lattitude`, `by longitude', or `has map coordinates`. | https://lux-front-tst.collections.yale.edu/view/results/places?q=%7B%22text%22%3A%22coordinates%22%2C%22_lang%22%3A%22en%22%7D&sq=coordinates

**Comment by roamye** [comment]
@clarkepeterf - based on this issue and https://github.com/project-lux/lux-frontend/issues/396, is there a reason why certain options do not show? I assume it is because it is not in the data, but wanted to confirm as I noticed Objects had some missing that do show in the data.

**Comment by roamye** [comment]
Approved by UAT.

Note: the order is incorrect, but does not block this ticket from moving forward. Needs a hot fix or a separate issue. let me know your preference. @clarkepeterf
